src/model/who.is.making.filing.model.ts

```diff
@@ -0,0 +1,6 @@
+export const WhoIsRegisteringKey = "who_is_registering";
+
+export enum WhoIsRegisteringType {
+    AGENT = "agent",
+    OVERSEAS = "overseas"
```

Comment:
Rather than 'OVERSEAS' wouldn't this be better named something like 'SOMEONE_ELSE' to match the option text on the screen?